packages/jsii/lib/transforms/deprecated-remover.ts

```diff
@@ -0,0 +1,891 @@
+import {
```

Comment:
I would recommend tests for such a large and complex code. I suppose it's coming soon.

Reply:
...granted I am not allocated to another project...